templates/galaxy/webhooks/subdomain_switcher/config.yml.j2

```
@@ -0,0 +1,5 @@
+id: subdomain_switcher
+type: masthead
+activate: true
+icon: fa-connectdevelop
```

```suggestion
icon: fa-flask
```

Wendi prefers fa-flask :)